src/core/lib/security/credentials/plugin/plugin_credentials.c

```diff
@@ -125,6 +140,7 @@ static bool plugin_get_request_metadata(grpc_exec_ctx *exec_ctx,
                                         grpc_closure *on_request_metadata,
                                         grpc_error **error) {
   grpc_plugin_credentials *c = (grpc_plugin_credentials *)creds;
```

Comment:
Could you add traces to track the begin/end of the sync/async request? It helps to debug potential deadlock issues mentioned here: https://github.com/grpc/grpc/pull/11553#issuecomment-332245169

Reply:
Done.